NEMO/admin.py

```diff
@@ -293,6 +298,16 @@ def save_model(self, request, obj, form, change):
 				obj.nonrequired_resource_set.set(form.cleaned_data["nonrequired_resources"])
 
 
+
+class AreaAdminForm(forms.ModelForm):
+	class Meta:
+		model = Area
+		fields = "__all__"
+	_area_calendar_color = forms.CharField(
+		required=False, max_length=9,
```

Comment:
I would also add `label="Area calendar color"` so it's consistent with the capitalization (when using a field that starts with _ it won't capitalize otherwise)